pkg/engine/transform/transform.go

```diff
@@ -104,6 +124,33 @@ func (t *Transformer) NormalizeForVMSSScaling(logger *logrus.Entry, templateMap
 	return nil
 }
 
+// NormalizeForK8sSLBScaling takes a template and removes elements that are unwanted in a K8s Standard LB cluster scale up/down case
+func (t *Transformer) NormalizeForK8sSLBScaling(logger *logrus.Entry, templateMap map[string]interface{}) error {
```

Comment:
looks like this isn't only being used for scale up (it's also used for upgrade) so let's make the name reflect that (ie. `NormalizeForK8sSLBScalingorUpgrading`, `NormalizeForK8sSLBUpdating` or something like that).

Reply:
Yea I was kind of debating about this one. `NormalizeForVMSSScaling` is currently being used by both scaling and upgrade. To reduce duplicate code, I'm going to rename these to `NormalizeForK8sSLBScalingOrUpgrade` and `NormalizeForK8sSLBScalingOrUpgrading`.